### PR TITLE
Lets update a bunch of our broken cogs to versions that work

### PR DESCRIPTION
**Pull request by DannyDB5544**
**This has honestly devolved into looking at the cogs we have and update the outdated ones, we're running stupidly old code in some cases. I'd make a joke, but it'd be lost on myself at this point. Feel free to yell at me in discord with a "What drugs are you on" If you feel this has gone into the sewers; but hey, Bots gotta be more than "what we need it to do in our server."**

Requested by Welk, A giveaway system styled on how he does giveaways.

~~The Trauma of Emotes is real~~

Leveler has been updated with hopefully a fix for badges (should be functional) as well as working backgrounds, better V3 data handling and some things that no one mentioned.

Mod has had what bugs existed quashed, as I have reverted us to the public version - a version we had LESS features than.

- The New Mod is under **ModEX**, in order to ensure that should it not work, we can unload and load standard mod.

Notsobot is now mostly functional again, for once, maybe, idk, the things a joke but I'd rather we had a functional joke.

 We also now have the public translation files for both Mod and Leveler! We.. Probably won't be needing them, but ive included them just in case.

**It should be noted that all we were doing is running stupidly out of date versions of both cogs. Updating all three to the public versions out there allows us to fix flaws and add things without breaking the rest of the cog/bot/server**

### CODE REVIEW

**Comment by DannyDB5544** [comment]
**DO NOT** Apply this PR to the bot **UNTIL** It is updated. The current install has a fault where it cant download any reqs, so we'd have to redo it anyway.

File: mod/names.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        """
+        author = ctx.author
+        guild = ctx.guild
+
+        if not user:
+            user = author
+
+        #  A special case for a special someone :^)
+        special_date = datetime(2016, 1, 10, 6, 8, 4, 443000)
+        is_special = user.id == 96130341705637888 and guild.id == 133049272517001216
```
[the remark this answers is not in the document]
Comment (DannyDB5544) [reply mod/names.py]: yeees

[the remark this answers is not in the document]
Comment (AgentBlackout) [reply mod/names.py]: By a Redbot-Dev apparently. Dribblers.

**Comment by DannyDB5544** [comment]
Last alteration Submitted, if i submit anything else thats not a bugfix consider this comment permission to call me a dunce

**Pull request merged**